### PR TITLE
Minimal changes required for OT 0.32

### PR DESCRIPTION
**Pull request by tylerbenson**
This seems backwards compatible with 0.31 since all changes are additional.

### CODE REVIEW

File: dd-trace-ot/src/main/java/datadog/opentracing/propagation/TagContext.java [hunk 1]
```diff
@@ -19,6 +19,16 @@ public TagContext(final Map<String, String> tags) {
...
 
+  @Override
+  public String toTraceId() {
+    return null;
+  }
+
+  @Override
+  public String toSpanId() {
+    return null;
+  }
```
Comment (mar-kolya) [inline dd-trace-ot/src/main/java/datadog/opentracing/propagation/TagContext.java]: Are `null`s actually expected return here?

Comment (tylerbenson) [reply dd-trace-ot/src/main/java/datadog/opentracing/propagation/TagContext.java]: Oops... looks like it should be an empty string: https://github.com/opentracing/specification/blob/master/rfc/trace_identifiers.md#specification-changes

File: dd-trace-ot/src/main/java/datadog/opentracing/scopemanager/ContextualScopeManager.java [hunk 2]
```diff
@@ -39,12 +44,17 @@ public Scope active() {
     return tlsScope.get();
   }
 
+  @Override
+  public Span activeSpan() {
+    return active().span();
```
Comment (realark) [inline dd-trace-ot/src/main/java/datadog/opentracing/scopemanager/ContextualScopeManager.java]: Any chance `active()` can return null?

Comment (tylerbenson) [reply dd-trace-ot/src/main/java/datadog/opentracing/scopemanager/ContextualScopeManager.java]: good catch.  fixed.

**Comment by leonardehrenfried** [comment]
I'm quite interested in OT 0.32 so looking forward to this!

File: dd-trace-ot/src/main/java/datadog/opentracing/propagation/TagContext.java (hunk 1 above)
Comment (tylerbenson) [reply dd-trace-ot/src/main/java/datadog/opentracing/propagation/TagContext.java]: Fixed.

**Comment by tylerbenson** [comment]
@leonardehrenfried I'm not sure when we will release this yet.  Would you be interested in helping us do some testing?  Here are the builds from this branch:
* https://21350-89221572-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/libs/dd-java-agent-0.27.0-SNAPSHOT.jar
* https://21350-89221572-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/libs/dd-trace-api-0.27.0-SNAPSHOT.jar
* https://21350-89221572-gh.circle-artifacts.com/0/home/circleci/dd-trace-java/libs/dd-trace-ot-0.27.0-SNAPSHOT.jar

**Comment by leonardehrenfried** [comment]
I'm hoping that I will get a chance to test this out in the next couple of
weeks.

Tyler Benson <notifications@github.com> schrieb am Fr., 29. März 2019,
19:16:

**Comment by Xyaren** [comment]
We really need this to use datadog with spring-kafka-opentracing, that only supports 0.32+

**Comment by kat1aka** [comment]
Hello. What's the status of this? Because our dependencies require the newer version of opentracing

**Pull request merged**